### PR TITLE
feat(cvs): Add support for custom controled vocabularies for non subj…

### PR DESCRIPTION
**Pull request by ioanpocol**
…ect fields (genre, place, ...)

### CODE REVIEW

**Comment by ioanpocol** [comment]
Depend on https://github.com/superdesk/superdesk-client-core/pull/319

File: server/data/content_types.json [hunk 1]
```diff
@@ -16,21 +16,20 @@
...
             "slugline": {"order": 1, "sdWidth": "half"},
             "priority": {"order": 2, "sdWidth": "quarter", "default": 6},
             "urgency": {"order": 3, "sdWidth": "quarter", "default": 5},
             "anpa_category": {"order": 4, "sdWidth": "full"},
-            "category": {"order": 5, "sdWidth": "half"},
-            "genre_service": {"order": 6, "sdWidth": "half"},
-            "subjects": {"order": 7, "sdWidth": "full"},
-            "places": {"order": 8, "sdWidth": "full"},
+            "category": {"order": 5, "sdWidth": "half", "schemafield": "subject", "required": true},
+            "genre_custom": {"order": 6, "sdWidth": "half", "schemafield": "genre", "required": true},
```
Comment (petrjasek) [inline server/data/content_types.json]: maybe we can add this `schemafield` to controlled vocabulary schema, seems like this won't change across the app, will it?

Comment (ioanpocol) [inline server/data/content_types.json]: Moved on vocabulary

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**